searchguard/rolesmapping.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/python3
+
+import requests
+import json
+import searchguard.settings as settings
+from searchguard.exceptions import *
+from searchguard.roles import check_role_exists
+
+
+def __process_data(role, properties):
+    """Private function to process API calls for the rolemapping module"""
+    create_sg_rolemapping = requests.put('{}/rolesmapping/{}'.format(settings.SEARCHGUARD_API_URL, role),
+                                         data=json.dumps(properties),
+                                         headers=settings.HEADER,
+                                         auth=settings.SEARCHGUARD_API_AUTH)
+
+    if create_sg_rolemapping.status_code == 201 or create_sg_rolemapping.status_code == 200:
```

Comment:
`if create_sg_rolemapping.status_code in (200, 201):`

Reply:
Either this option or the one I mentioned in my [comment](https://github.com/ByteInternet/searchguard-python/pull/21#discussion_r239467524) I'm fine with.

Reply:
I prefer more specific status codes

Reply:
In that case, the `if create_sg_rolemapping.status_code in (200, 201):` is better.